### ISSUE TITLE
tfrecord support in local mode?

### ISSUE THREAD

**Issue by gitrc** [aws-samples/amazon-sagemaker-local-mode#21]
It appears that only 'Line' and 'RecordIO' split_type are supported in local mode.  Is it possible to support TFRecord?  Thank you.

**Comment by eitansela** [comment]
Hello @gitrc , can I please have a reference from where did you get the information that only 'Line' and 'RecordIO' split_type are supported in local mode?
Thanks!

**Comment by gitrc** [comment]
@eitansela  sure, https://github.com/aws/sagemaker-python-sdk/blob/9f400bd0f76a6c95807382314e4ad5074c4d1b28/src/sagemaker/local/data.py#L60

**Comment by eitansela** [comment]
Yes you are right. You can open an issue in SageMaker Python SDK GitHub repo, and to expedite it, you can write me a private mail stating what is the company you work for, so I can introduce you to the Machine Learning Specialist working with you.

**Comment by gitrc** [comment]
Thank you, I created https://github.com/aws/sagemaker-python-sdk/issues/3626 and will attempt to contact you directly via other means.
